llvm/lib/Support/Base64.cpp

```diff
@@ -90,3 +90,180 @@ llvm::Error llvm::decodeBase64(llvm::StringRef Input,
   }
   return Error::success();
 }
+
```

Comment:
Existing encode/decode logic works well for char arrays. We need something that will work for std::byte arrays.

Thanks